value/annotations/pom.xml

```diff
@@ -24,17 +24,16 @@
     <version>HEAD-SNAPSHOT</version>
   </parent>
 
-  <groupId>com.google.auto.value</groupId>
   <artifactId>auto-value-annotations</artifactId>
   <version>HEAD-SNAPSHOT</version>
   <name>AutoValue Annotations</name>
   <description>
-    Immutable value-type code generation for Java 1.6+.
+    Immutable value-type code generation for Java 1.7+.
   </description>
   <url>https://github.com/google/auto/tree/master/value</url>
 
   <properties>
-    <java.version>1.7</java.version>
+    <java.version>1.8</java.version>
```

Comment:
https://github.com/google/auto/blob/master/value/userguide/index.md#what-java-versions-does-it-work-with

Reply:
I don't think this change is accurate; that link says that the compiler must be a JDK 8+ compiler but that compilation can still be source/target 7. This would change the source/target versions, I believe.

Reply:
@cgdecker you're right. Reverted.